tests/builder/features_test.py

```diff
@@ -147,6 +147,46 @@ def test_feature_name_automatic(tmpdir, ufo_module):
         };
         # automatic
         sub g by g.ss01;
+
+        } ss01;
+    """
+    )
+
+    font, rtufo = roundtrip(ufo, tmpdir, ufo_module)
+
+    # Check code in Glyphs font
+    gs_feature = font.features[0]
+    assert gs_feature.automatic
+    assert gs_feature.code.strip() == "sub g by g.ss01;"
+    assert gs_feature.notes.strip() == "Name: Alternate g"
```

Comment:
This assertion fails:

```bash
        assert gs_feature.code.strip() == "sub g by g.ss01;"
>       assert gs_feature.notes.strip() == "Name: Alternate g"
E       AssertionError: assert '' == 'Name: Alternate g'
E         - Name: Alternate g

tests/builder/features_test.py:161: AssertionError
```

I have no idea why it doesn't work. When I convert a similar file with `ufo2glyphs`, everything looks fine. If you see some stupid error that I made, let me know ;)